common/app/model/GarnettDesignType.scala

```diff
@@ -0,0 +1,10 @@
+package model
+
+import com.gu.contentapi.client.utils.DesignType
+
+object GarnettDesignType {
```

Comment:
I don't think we should mention Garnett for anything not temporary. In a year, most of the devs won't remember what Garnett means

Reply:
Yep, I couldn't call this object `DesignType` as it obscured the imported `DesignType`